MultiplayerMod/Core/Logging/Logger.cs

```diff
@@ -19,6 +21,7 @@ public class Logger {
     public void Trace(Func<string> messageProvider) => Log(LogLevel.Trace, messageProvider());
 
     private void Log(LogLevel level, string message) {
+        if (level < LogLevel) return;
```

Comment:
Let's use line breaks for `if`s, I guess I didn't add the rule for the autoformat, if there any.